src/index.js

```diff
@@ -12,3 +12,6 @@ if (typeof window !== 'undefined') {
 }
 
 export const ver = '__VERSION__';
+export function setupCanvasMock() {
+  mockWindow(global.window);
+}
```

Comment:
Maybe this function could accept an argument `window`, defaulting to `global.window`. This would allow mocking the canvas on a jsdom window that is not the global object.

```suggestion
export function setupCanvasMock(window) {
  mockWindow(window || global.window);
}
```

Reply:
I added #100 to assist with this suggestion.

Reply:
sounds good to me :+1: